lib/Mojolicious/resources/templates/mojo/debug.html.ep

```diff
@@ -100,18 +98,22 @@
     </header>
     <script>
       function mojoDrawer(handle, drawer) {
-        $(handle).on('click', function () {
-          $(drawer).slideToggle('slow');
-          const text = $(handle + ' div.tap').text();
+        const handleEl = document.querySelector(handle);
+        handleEl.addEventListener('click', () => {
+          drawerEl = document.querySelector(drawer);
+          let text = handleEl.innerText;
           text = text == "tap for more" ? "tap for less" : "tap for more";
-          $(handle + ' div.tap').text(text);
+          const divTap = document.querySelector(handle + ' div.tap');
+          if (divTap !== null) {
+            divTap.innerText = text;
```

Comment:
I like to format these as:
```js
if (divTap !== null) divTap.innerText = text;
```
Instead of blocks.

Reply:
Done.